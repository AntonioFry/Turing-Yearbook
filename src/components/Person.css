.person {
  background-color: white;
  border-radius: 20px;
  padding: 5px;
}

img {
  border-radius: 10px;
  max-width: 150px;
  max-height: 150px;
}