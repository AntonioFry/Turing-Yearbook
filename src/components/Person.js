import React from 'react';
import './Person.css';

const Person = (props) => {
  return (
    <div>
      <h2>{props.name}</h2>
      <img src={props.photo} alt="a bird"/>
      <p className="quote">{props.quote}</p>
      <p>{props.superlative}</p>
    </div>
  )
}

export default Person;
