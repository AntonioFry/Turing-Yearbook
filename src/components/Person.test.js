import React from 'react';
import { shallow, mount } from 'enzyme';
import Person from './Person';

describe('Card', () => {

  var data;
  beforeEach(function() {
    data = { id: 1, name: 'name', photo: 'image', quote: 'quote', superlative: 'superlative' }
  });

  it('match the snapshot with all the data passed in correctly', () => {
    const wrapper = shallow(
      <Person
        key={data.key}
        photo={data.photo}
        name={data.name}
        quote={data.quote}
        superlative={data.superlative}
      />
    )

    expect(wrapper).toMatchSnapshot();
  });
});