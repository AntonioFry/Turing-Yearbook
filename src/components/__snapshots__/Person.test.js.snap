// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card match the snapshot with all the data passed in correctly 1`] = `
<div
  className="person"
>
  <h2>
    name
  </h2>
  <img
    alt="a bird"
    src="image"
  />
  <p
    className="quote"
  >
    quote
  </p>
  <p>
    superlative
  </p>
</div>
`;
